modules/gateways/callback/blockonomics.php

```diff
@@ -36,51 +37,53 @@
 $secret_value = $blockonomics->getCallbackSecret();
 
 if ($secret_value != $secret) {
-	$transactionStatus = $_BLOCKLANG['error']['secret'];
-	$success = false;
+    $transactionStatus = $_BLOCKLANG['error']['secret'];
+    $success = false;
 
-	echo $transactionStatus;
-	die();
+    echo $transactionStatus;
+    die();
 }
 
 $order = $blockonomics->getOrderByAddress($addr);
 $invoiceId = $order['order_id'];
 $bits = $order['bits'];
 
 // If this is test transaction, generate new transaction ID
-if($txid == 'WarningThisIsAGeneratedTestPaymentAndNotARealBitcoinTransaction') {
-	$txid = 'WarningThisIsATestTransaction_' . $invoiceId;
+if ($txid == 'WarningThisIsAGeneratedTestPaymentAndNotARealBitcoinTransaction') {
+    $txid = 'WarningThisIsATestTransaction_' . $invoiceId;
 }
 
 $confirmations = $blockonomics->getConfirmations();
 
 $blockonomics_currency_code = $order['blockonomics_currency'];
 $blockonomics_currency = $blockonomics->getSupportedCurrencies()[$blockonomics_currency_code];
-if($blockonomics_currency_code=='btc'){
-	$subdomain = 'www';
-}else{
-	$subdomain = $blockonomics_currency_code;
+if ($blockonomics_currency_code == 'btc') {
+    $subdomain = 'www';
+} else {
+    $subdomain = $blockonomics_currency_code;
 }
-if($status < $confirmations) {
-	$invoiceNote = "<b>".$_BLOCKLANG['invoiceNote']['waiting']." <img src=\"img/".$blockonomics_currency_code.".png\" style=\"max-width: 20px;\"> ".$blockonomics_currency->name." ".$_BLOCKLANG['invoiceNote']['network']."</b>\r\r" .
-		$blockonomics_currency->name." transaction id:\r" .
-		"<a target=\"_blank\" href=\"https://".$subdomain.".blockonomics.co/api/tx?txid=$txid&addr=$addr\">$txid</a>";
 
-	$blockonomics->updateOrderInDb($addr, $txid, $status, $value);
-	$blockonomics->updateInvoiceNote($invoiceId, $invoiceNote);
+$systemUrl = App::getSystemUrl();
```

Comment:
There is an existing blockonomics method that can be used here: `$systemUrl = $blockonomics->getSystemUrl();`

Reply:
Right, I was unaware of that one. 

However, it conserns me why there is a method wrapping an existing and official one? I just reviewed the method blockonomics::getSystemUrl which is running his own way to get the URL with no need, the App::getSystemUrl is the official method to get the URL. (see WHMCS files feeds/domainchecker.php and feeds/productpricing.php)

In fact, my recommendation is to replace that blockonomics::getSystemUrl and implement the official one, thought this is matter for a different issue/branch.

Reply:
I have created a new issue for this:
https://github.com/blockonomics/whmcs-bitcoin-plugin/issues/95

For uniformity until then, let's use: `$blockonomics->getSystemUrl();` in this pull request.